src/Symfony/Component/Asset/Preload/PreloadManagerInterface.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Asset\Preload;
+
+/**
+ * Manages resources to preload according to the W3C "Preload" specification.
+ *
+ * @see https://www.w3.org/TR/preload/
+ *
+ * @author Kévin Dunglas <dunglas@gmail.com>
+ */
+interface PreloadManagerInterface
```

Comment:
This extension point allows to replace the manager by a smarter one (for instance if you want to create one giving access to the list of resources added to change them programmatically).
IMO it doesn't hurt and allow to easily replace the global app manager to add custom behaviors.